pkg/models/plan_cr_tdl.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"time"
+)
+
+// PlanCrTdl represents CRs and TDLs related to a model plan
+type PlanCrTdl struct {
+	BaseStruct
+	ModelPlanRelation
+	IDNumber         string     `json:"idNumber" db:"id_number"`
+	DateInitiated    *time.Time `json:"dateInitiated" db:"date_initiated"`
+	Title            string     `json:"title" db:"title"`
+	OptionalComments string     `json:"optionalComments" db:"optional_comments"`
```

Comment:
Non blocking, but it's more in line with existing tables to call the OptionalComments field note.

Reply:
@StevenWadeOddball Changes made